applications/MappingApplication/tests/blade_mapping_test.py

```diff
@@ -3,7 +3,7 @@
 default_data_comm = KM.Testing.GetDefaultDataCommunicator()
 if default_data_comm.IsDistributed():
     from KratosMultiphysics import mpi as KratosMPI
-    from KratosMultiphysics.MappingApplication import MPIExtension as MappingTrilinosExtension
+    from KratosMultiphysics.MappingApplication import TrilinosExtension as MappingTrilinosExtension
```

Comment:
nooo, please dont change this :D

I know it technically still depends on trilinos, but the plan is to replace with the Kratos-CSR matrix to drop the dependency entirely!

Reply:
Ok... but then we should change the others to MPI extension instead of Trilinos, otherwise is super confusing

Reply:
Reverted the changes :+1: